setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+conda create -n xnat_downloader_env python=2.7
+source activate xnat_downloader_env
+conda install -c conda-forge dcm2niix
+python setup.py
```

Comment:
should this be `python setup.py install`? I got an error when I tried to run this as is.

Reply:
@jdkent good catch! just fixed that and the other requested changes